packages/devtools/src/devtoolsdriver.js

```diff
@@ -133,8 +133,8 @@ export default class DevToolsDriver {
         page.setDefaultTimeout(this.timeouts.get('pageLoad'))
     }
 
-    getPageHandle () {
-        if (this.currentFrame) {
+    getPageHandle ({ frame = false } = {}) {
```

Comment:
I guess this way it is extendable but not sure how many properties we will add to this method, therefor I would suggest to just have a parameter called `isInIFrame` which would simplify this a bit

Reply:
ok, I'll rename `frame` to `isInFrame` instead of `isInIFrame` because context may be not necessary an `iframe`

Reply:
@christian-bromann done